docs/050-breaking-changes.rst

```diff
@@ -0,0 +1,85 @@
+********************************
+Solidity v0.5.0 Breaking Changes
+********************************
+
+This section highlights the main breaking changes introduced in Solidity
+version 0.5.0, along with the reasoning behind the changes and how to update
+affected code.
+For the full list check
+`the release changelog <https://github.com/ethereum/solidity/releases>`_.
+
+Semantic Only Changes
+=====================
+
+This section lists the changes that are semantic-only, thus potentially
+hiding new and different behaviors in previous code.
+
+* Signed right shift now uses proper arithmetic shift, i.e. rounding towards
+  negative infinity. Signed and unsigned shift will have dedicated opcodes in
+  Constantinople, and are emulated by Solidity for the moment.
+
+* The ``continue`` statement in a ``do...while`` loop now jumps to the
+  condition, which is the common behavior in such cases. It used to jump to the
+  loop body.
```

Comment:
Can you add another sentence that explains the same things without using the terms "jump" (i.e. "breaks out of the loop if the condition is false" or something like this)?